src/views/Csv.vue

```diff
@@ -0,0 +1,60 @@
+<template>
+<div>
+  <h1>json to csv</h1>
+  <input type="textarea" rows="4" cols="50" v-model="json">
+  <!-- You can also provide the following data in the input field in json form
+  [
+    {
+        "Column 1": "1-1",
+        "Column 2": "1-2",
+        "Column 3": "1-3",
+        "Column 4": "1-4"
+    },
+    {
+        "Column 1": "2-1",
+        "Column 2": "2-2",
+        "Column 3": "2-3",
+        "Column 4": "2-4"
+    },
+    {
+        "Column 1": "3-1",
+        "Column 2": "3-2",
+        "Column 3": "3-3",
+        "Column 4": "3-4"
+    },
+    {
+        "Column 1": 4,
+        "Column 2": 5,
+        "Column 3": 6,
+        "Column 4": 7
+    }
+  ] -->
+  <button v-on:click="convert">Convert</button>
```

Comment:
We can use shorthand `@click` instead of `v-on:`

Reply:
Used shorthand.
Thank you sir for reviewing my code.